resources/lang/esp.ini

```diff
@@ -0,0 +1,112 @@
+command.flip.try = "Intentando voltear el portapapeles por {%0}"
+command.flip.success = "Portapapeles invertido con éxito"
+command.rotate.try = "Intentando rotar el portapapeles en {%0} grados"
+command.rotate.success = "Portapapeles rotado con éxito"
+command.history.cleared = "Historial borrado"
+command.listchunks.found = "{%0} fragmentos encontrados en la selección"
+command.size = "Tamaño de selección"
+; selection
+selection.pos1.set = "Posición 1 establecida en X: {%0} Y: {%1} Z: {%2}"
+selection.pos2.set = "Posición 2 establecida en X: {%0} Y: {%1} Z: {%2}"
+; session
+session.undo.none = "Nada que deshacer"
+session.undo.left = "Te quedan {%0} acciones para deshacer"
+session.redo.none = "Nada que rehacer"
+session.redo.left = "Te quedan {%0} acciones de rehacer"
+session.brush.added = "Agregado {%0} a la sesión"
+session.brush.deleted = "Eliminado {%0} (UUID {%1})"
+session.brush.removed = "Eliminado {%0} (UUID {%1})"
+session.language.set = "Establecer correctamente el idioma en {%0}"
+session.language.notfound = "Idioma {%0} no encontrado, restableciendo a los valores predeterminados"
+; task
+task.copy.success = "La copia asíncrona se realizó correctamente, tomó {%0}, copió {%1} bloques de {%2}".
+task.count.success = "El análisis asincrónico tuvo éxito, tomó {%0}"
+task.count.result = "{%0} bloques encontrados en un total de {%1} bloques"
+task.fill.success = "El llenado asíncrono tuvo éxito, tomó {%0}, {%1} bloques de {%2} cambiado".
+task.replace.success = "Reemplazo asíncrono exitoso, tomó {%0}, {%1} bloques de {%2} cambiado".
+task.revert.undo.success = "Deshacer asíncrono exitoso, tomó {%0}, {%1} bloques de {%2} cambiado".
+task.revert.redo.success = "Rehacer asíncrono tuvo éxito, tomó {%0}, {%1} bloques de {%2} cambiado".
+; flags
+flags.keepexistingblocks = "Mantener bloques existentes"
+flags.keepair = "Mantener el aire"
+flags.hollow = "Hueco"
+flags.hollowclosed = "Hueco con extremos cerrados"
+flags.natural = "Natural"
+; tools
+; wand tool
+tool.wand = "Varita mágica"
+tool.wand.lore.1 = "Haga clic izquierdo en un bloque para establecer la posición 1 de una selección"
+tool.wand.lore.2 = "Haga clic derecho en un bloque para establecer la posición 2 de una selección"
+tool.wand.lore.3 = "Use //togglewand  para alternar su funcionalidad"
+tool.wand.disabled = "La herramienta de varita está deshabilitada. Use //togglewand para volver a habilitarla"
+tool.wand.setenabled = "¡La herramienta de varita ahora es {% 0}!"
+; debug tool
+tool.debug = "Herramienta de depuración"
+tool.debug.lore.1 = "Haga clic izquierdo en un bloque para obtener información"
+tool.debug.lore.2 = "Como el nombre y los valores de daño de un bloque"
+tool.debug.lore.3 = "Use //toggledebug para alternar su funcionalidad"
+tool.debug.disabled = "La herramienta de depuración está deshabilitada. Use //toggledebug para volver a habilitarla"
+tool.debug.setenabled = "¡La herramienta de depuración ahora es {%0}!"
```

Comment:
Just a question, what is this upside down "!" - have seen this quite often already :P

Reply:
They are used to indicate surprise, amazement, joy, supplication :)

Reply:
Pretty cool, thanks :D